app/ergoCubEmotions/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+################################################################################
+#                                                                              #
+# Copyright (C) 2023 Fondazione Istitito Italiano di Tecnologia (IIT)          #
+# All Rights Reserved.                                                         #
+#                                                                              #
+################################################################################
+
+project(ergoCubEmotions)
+cmake_minimum_required(VERSION 3.12)
+
+find_package(YARP 3.7.2 REQUIRED)
```

Comment:
I would avoid putting two `find_package(YARP)` so I think it is the case of move it in the root CMakeLists.txt, this will make YARP mandatory for having just `YARP_CONTEXTS_INSTALL_DIR` but I didn't find any other solution

cc @traversaro

Reply:
I agree with @Nicogene . Unless there is a specific reason to have theme here, I would also get rid of the `project` and `cmake_minimum_required` calls, as thy would only mess the `PROJECT_` variables and the CMake policy used in this subdirectory, with no benefit.